tests/end2end/MeetingSeries_security.js

```diff
@@ -0,0 +1,100 @@
+import { E2EGlobal } from './helpers/E2EGlobal'
+import { E2EApp } from './helpers/E2EApp'
+import { E2ESecurity } from './helpers/E2ESecurity'
+import { E2EMeetingSeries } from './helpers/E2EMeetingSeries'
+
+describe('MeetingSeries Security', function () {
+    const insertMeetingSeriesMethod = "meetingseries.insert";
+    const removeMeetingSeriesMethod = "workflow.removeMeetingSeries";
+    const updateMeetingSeriesMethod ="meetingseries.update";
+
+    beforeEach("goto start page and make sure test user is logged out", function () {
+        E2EApp.gotoStartPage();
+        E2EApp.loginUser();
+        expect(E2EApp.isLoggedIn()).to.be.true;
+    });
+
+    before("reload page and reset app", function () {
+        E2EApp.resetMyApp(true);
+        E2EApp.launchApp();
+    });
+
+    it('can not insert a new MeetingSerie if not logged in', function () {
+        let aProjectName = "Hacker Project #1";
+        let aMeetingName = "Hacker Meeting #1";
+        E2EApp.logoutUser();
+        expect(E2EApp.isLoggedIn()).to.be.false;
+
+        E2ESecurity.expectMethodToExist(insertMeetingSeriesMethod);
+        E2ESecurity.replaceMethodOnClientSide(insertMeetingSeriesMethod);
+        
+        let noOfMeetingSeries = server.call('e2e.countMeetingSeriesInMongDB');
```

Comment:
I would prefer the name `meetingSeriesCount` which is easier to pronounce and you do not have to think about the meaning of `no` which seems to be used as an abbreviation of `number`.